specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2025-07-01/SummaryLogs.json

```diff
@@ -0,0 +1,653 @@
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SummaryLogsNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SummaryLogsRetryBin"
+            },
+            "description": "The parameters required to retry a Summary rule bin."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted response definition (long-running operation).",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              },
+              "Location": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RuleDefinition": {
+      "type": "object",
+      "properties": {
+        "query": {
+          "type": "string",
+          "description": "Summary rule query."
+        },
+        "binSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Scheduled window in minutes. Allowed values: 20, 30, 60, 120, 180, 360, 720, 1440."
+        },
+        "binDelay": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum delay in seconds before bin processing."
+        },
+        "binStartTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time (UTC) when Summary rule execution starts."
+        },
+        "timeSelector": {
+          "type": "string",
+          "enum": [
+            "TimeGenerated"
+          ],
+          "description": "The time cursor used in Summary rules bins processing, e.g. TimeGenerated.",
+          "x-ms-enum": {
+            "name": "TimeSelectorEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "TimeGenerated",
+                "description": "TimeGenerated."
+              }
+            ]
+          }
+        },
+        "destinationTable": {
```

Comment:
>destinationTable [](http://example.com/codeflow?start=9&length=16)

can this be an enum with a list of possible values ?

Reply:
Its a mere single destination table.

Also applies:
Contract already being used by customers and passed historic review by ARM during private preview.
https://github.com/Azure/azure-rest-api-specs-pr/pull/15528

Contract and names were also devised by the PM team.